cmd/outputs.go

```diff
@@ -19,30 +19,46 @@ import (
 	"github.com/grafana/xk6-output-prometheus-remote/pkg/remotewrite"
 )
 
+// builtinOutput marks the available builtin outputs.
+//
+//go:generate enumer -type=builtinOutput -trimprefix builtinOutput -transform=kebab -output builtin_output_gen.go
+type builtinOutput uint32
+
+const (
```

Comment:
>enums decrease code reading/maintaining abilities

The reason is to reduce the probability of introducing bugs in the future by changing the list of builtin outputs (adding or removing them). In this way, we can rely on a single source of truth across the code more than directly on strings.

>and it seems like the builtin prefix could be omitted

Do you mean only from the items' names or also from the type? Because having an `output` type when we already have the `output` package and `output.Output` type could create confusion if there isn't an element to differentiate them.

Reply:
Honestly, as someone who has never used `enumer` before, I have to admit that the generated code looks a bit overwhelming. But as suggested before, I'm _generally_ in favor or reducing chances to introduce errors by reducing duplicity, and as `enumer` seems to be already used in the project, I'd say it looks good enough to be merged to me.